lib/git_reflow/git_server/git_hub/pull_request.rb

```diff
@@ -63,7 +63,19 @@ def comments
         end
 
         def last_comment
-          "#{comments.last.body.inspect}"
+          if comments.last.nil?
+            ""
+          else
+            "#{comments.last.body.inspect}"
+          end
+        end
+
+        def approved?
```

Comment:
Haha, my head is spinning.  No more sample code from me.  Let's consolidate all the logic for this into the original base `PullRequest#approved?` method and just check the `reflow.git-server` config setting for what server they're using and only check last comment if it's `... =~ /github/i`.